lib/model/binding.dart

```diff
@@ -303,10 +303,12 @@ class LinuxDeviceInfo implements BaseDeviceInfo {
 class PackageInfo {
```

Comment:
```
binding: add packageName to the packageInfo class in the binding class 
made changes to access packageName from the PackageInfo class.
remove getAppBundleId() from Zulip Binding class
Add packageName variable to PackageInfo class
Fixes #407
```

The commit claims that `getAppBundleId()` got removed, but it is not present in the commit itself. For changes that address reviews on a previous revision of the commit, the PR thread is more appropriate.

The commit summary and the "Fixes" line also need to be updated. This commit doesn't fix #407. We should move this line to the next commit where we start using `packageName`.

For the commit message, changes that one can tell from reading the diffs generally don't need to be reiterated: "made changes to access packageName from the PackageInfo class"
"Add packageName variable to PackageInfo class".

Please review Zulip's commit discipline linked in the README https://github.com/zulip/zulip-flutter?tab=readme-ov-file#submitting-a-pull-request